libs/blocks/section-metadata/section-metadata.css

```diff
@@ -203,6 +203,17 @@ main > .section[class*='-up'] > .content {
   display: grid;
 }
 
+/* mobile only */
+@media (max-width: 600px) {
+  .section.reverse-mobile > div:nth-child(1) {
+    order: 2;
+  }
+
+  .section.reverse-mobile > div:nth-child(2) {
+    order: 1;
+  }
+}
+
```

Comment:
Your description says this will only work for `two-up`. But according to this it will work with any of the ups. 

Do we want to scope it to  `.two-up` or leave it open like it is? 

I guess my main thinking is. What's stopping authoring from trying this on three-up, four-up, etc., then submitting bug tickets since it only orders the first two children.

Reply:
Ah, yes, Good eye mate. I've addressed this to scope as described. 
I don't want this option available on any more `ups` then a `two-up`, otherwise it'll get real hard to manage orders.